build/_k-standard-goals.shade

```diff
@@ -40,6 +40,14 @@ default NUGET_FEED = 'https://api.nuget.org/v3/index.json'
   }
 
   Log.Info("Build v2: " + IsBuildV2);
+
+  // It looks like it isn't possible to do this in .travis.yml because we only want
+  // to do it on Linux and I couldn't find an easy way to do that kind of selective
+  // environment variable set.
+  if(E('TRAVIS') == "true" && CanBuildForLinux)
```

Comment:
`if (` formatting. Also, does this need to work for Mac - does `CanBuildForLinux` work in that scenario?